### PR TITLE
update repository and bugs url

### PR DESCRIPTION
**Pull request by lholmquist**
@abstractj updating the urls for git and bugs to not be pointing to @bobmcwhirter 's repos anymore

:eyes:

### CODE REVIEW

**Comment by lholmquist** [comment]
oops, forgot the links in the readme

**Comment by lholmquist** [comment]
@abstractj i removed the documentation link from the readme,  since it doesn't look like keycloak has a github.io page.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,5 @@ Embodies JSON Web Token bits and checking for roles.
 
 # Resources
 
-* [GitHub](http://github.com/bobmcwhirter/keycloak-auth-utils)
-* [Documentation](http://bobmcwhirter.github.io/keycloak-auth-utils)
```
Comment (abstractj) [inline README.md]: @lholmquist for documentation would make sense to migrate what Bob already did here https://github.com/bobmcwhirter/keycloak-auth-utils/tree/gh-pages ?

Comment (lholmquist) [inline README.md]: that would make sense,  do you need to turn on gh-pages or something then,  not sure what would be needed

Comment (abstractj) [inline README.md]: @lholmquist done https://github.com/keycloak/keycloak-nodejs-auth-utils/tree/gh-pages

Comment (lholmquist) [inline README.md]: neat, so what would the url be then?

Comment (abstractj) [inline README.md]: @lholmquist http://keycloak.github.io/keycloak-nodejs-auth-utils. I believe that if we are cool with that, we need one step before. Pretty much migrate Bob's documentation.

**Comment by lholmquist** [comment]
@abstractj ok,  i think we are ready to go on this now,  updated the documentation link with the new gh-pages

**Comment by abstractj** [comment]
landed, thank you

**Pull request merged**